crates/turborepo-lib/src/config/env.rs

```diff
@@ -80,7 +81,19 @@ impl ResolvedConfigurationOptions for EnvVars {
 
         let force = self.truthy_value("force").flatten();
         let remote_only = self.truthy_value("remote_only").flatten();
+        if remote_only.is_some() {
+            warn!(
+                "TURBO_REMOTE_ONLY is deprecated and will be removed in a future major version. \
+                 Use TURBO_CACHE=remote:rw"
+            );
```

Comment:
@anthonyshew high priority we have to set this on vercel before we ship this. I can spin this up quick